models/settings_helpers.go

```diff
@@ -65,6 +66,11 @@ type ChangeSettingsParams struct {
 	// not url.URL to keep username and password
 	AlertManagerURL       string
 	RemoveAlertManagerURL bool
+
+	// Enable Secutiry Threat Tool
+	EnableSTT bool
+	// Disable Secutiry Threat Tool
+	DisableSTT bool
```

Comment:
Why we need two fields for STT config? To change default behavior in the future?

Reply:
In gRPC/JSON request, we can't differentiate between `{"stt": false}`, `{"stt": null}`, and `{}`. So we have to use something else if zero value is useful. The biggest problem is boolean flags, so we use command verbs in requests: enable_stt, disable_stt. Both can't be present, but both can be absent.

See also other resolved discussions in that PR and in API tests

Reply:
Oh, it makes sense, thanks!

Reply:
Because of go's default values. The same way we have them for telemetry, we need fields to explicitly enable/disable a function. 
Suppose we only use enable. If the request comes with the default values, it would be false and we would be disabling the feature.

Reply:
Oh, and another reason: that API is somewhat special because there we expect users to send only what they want to change, not the whole state/model. Request `{}` is valid request for that API.

Reply:
💜